core/test/Cardano/Address/Style/ShelleySpec.hs

```diff
@@ -1235,6 +583,18 @@ prop_roundtripTextEncodingPointer encode' decode addXPub ptr discrimination =
     address = pointerAddress discrimination (PaymentFromKey addXPub) ptr
     result  = decode (encode' address)
 
+goldenText :: String -> Text -> Golden Text
+goldenText name actualOutput =
+  Golden {
+    output = actualOutput,
+    encodePretty = show,
```

Comment:
What's this field for - is this not the proper place to apply pretty printing?

Reply:
Not exactly, it's for displaying it in the output I'd assume. According to the type `str -> String` at least.